x-pack/filebeat/input/awscloudwatch/cloudwatch.go

```diff
@@ -183,6 +202,12 @@ func (p *cloudwatchPoller) receive(ctx context.Context, logGroupIDs []string, cl
 
 		// Advance to the next time span
 		startTime, endTime = endTime, clock().Add(-p.config.Latency)
+
+		// Store the last sync timestamp
+		err := p.stateHandler.StoreState(p.config, StorableState{LastSyncEpoch: endTime.UnixMilli()})
+		if err != nil {
+			p.log.Errorf("error storing state: %v, next sync(s) will continue without state storage support", err)
+		}
```

Comment:
You could make this one line:

```
if err := p.stateHandler.StoreState(p.config, StorableState{LastSyncEpoch: endTime.UnixMilli()}); err != nil {
	p.log.Errorf("error storing state: %v, next sync(s) will continue without state storage support", err)
}
```

Reply:
Yup, did it with the latest commit -https://github.com/elastic/beats/commit/86c216643f6c72b7eb6c6f536db45d100777b67f